routes/logs.js

```diff
@@ -3,8 +3,7 @@ const router = express.Router();
 const logs = require("../controllers/logs");
 const authenticate = require("../middlewares/authenticate");
 const { authorizeUser } = require("../middlewares/authorization");
-const { ROLES } = require("../constants/users");
 
-router.get("/:type", authenticate, authorizeUser(ROLES.SUPER_USER), logs.fetchLogs);
+router.get("/:type", authenticate, authorizeUser("superUser"), logs.fetchLogs);
```

Comment:
Are we sure that, this is the correct role for the super user ??
At most places I guess `super_user` is the role and can we please import it from a constant from `constants/roles.js`

Reply:
The parameter for authorizeUser can have these 3 values : superUser, appOwner, default (Check out https://github.com/Real-Dev-Squad/website-backend/blob/15e6242972b7f674af869b618ed42a1c5ae080fd/middlewares/authorization.js#L8)

There's an existing ticket for moving this parameter value to a constant: https://github.com/Real-Dev-Squad/website-backend/issues/633

Reply:
`superUser` is a Legacy role and will not be used further I guess, and also the file you are pointing to, the actual value is `super_user` only.

Reply:
We can use the LEGACY ROLE from constants for now, so that things work now. After the middleware changes, we will take care of it in refactoring

Reply:
As discussed we can use superUser legacy role currently and #582 will remove this later